File: aspnet/web-forms/overview/older-versions-security/membership/user-based-authorization-vb/samples/sample13.vb
Protected Sub FilesGrid_RowDeleting(ByVal sender As Object, ByVal e As System.Web.UI.WebControls.GridViewDeleteEventArgs)Handles FilesGrid.RowDeleting
 Dim fullFileName As String = FilesGrid.DataKeys(e.RowIndex).Value.ToString()
 FileContents.Text = String.Format("You have opted to delete {0}.", fullFileName)

 ' To actually delete the file, uncomment the following line
 ' File.Delete(fullFileName)
End Sub

File: aspnet/web-forms/overview/older-versions-security/membership/user-based-authorization-cs/samples/sample13.cs
protected void FilesGrid_RowDeleting(object sender, GridViewDeleteEventArgs e)
{
	string fullFileName = FilesGrid.DataKeys[e.RowIndex].Value.ToString();
	FileContents.Text = string.Format("You have opted to delete {0}.", fullFileName);

	// To actually delete the file, uncomment the following line
	// File.Delete(fullFileName);
}